images/linux/scripts/installers/julia.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+################################################################################
+##  File:  julia.sh
+##  Desc:  Installs Julia, and adds Julia to the path
+################################################################################
+
+# Source the helpers for use with the script
+source $HELPER_SCRIPTS/document.sh
+
+# This function fetches the latest Julia release from the GitHub API
+# Based on https://gist.github.com/lukechilds/a83e1d7127b78fef38c2914c4ececc3c
+function GetLatestJuliaRelease () {
+    curl --silent "https://api.github.com/repos/julialang/julia/releases/latest" |
```

Comment:
I don't think this gets you the release with highest version number (I assume that's what you are after), IIRC it just returns the latest (as in time) release.

Reply:
@fredrikekre is correct. So if we release 1.6.0, and the next day release 1.5.1, the GitHub API (if I understand correctly) will return 1.5.1 as the "latest release."

I think the correct thing to do is for the JuliaLang community to host a URL that simply redirects to the GitHub release of the **stable** Julia release with the highest version number. So e.g. right now it would redirect to: https://github.com/JuliaLang/julia/releases/tag/v1.3.1

Maybe @staticfloat or @maleadt could help with setting up this URL? I think it should be relatively straightforward:
1. Use the GitHub API to get the list of all releases.
2. Filter out the pre-releases, release candidates, etc.
3. Sort the remaining releases by SemVer version number in decreasing order.
4. Pick the release with highest version number.

We would run this computation once per day as a cron job, and save the result somewhere, and then that result is hosted at a JuliaLang.org URL.

Reply:
@DilumAluthge see https://github.com/JuliaLang/julia/issues/33817

Reply:
It looks like that (if it is ever implemented) would provide all of the versions? So we’d still have to go through and figure out what the latest version is?

Reply:
Yes.

Alternatively, the steps you outlined could be added to this build script.

@al-cheb Do you know if `jq` is available in the build environment?